rapid7_insightvm_cloud/help.md

```diff
@@ -0,0 +1,1002 @@
+# Description
```

Comment:
All of the examples in here need to be anonymized/conform to the examples in the examples document. I'll send it to you.